packages/guides/src/Nodes/DocumentTree/DocumentEntryNode.php

```diff
@@ -38,27 +36,17 @@ public function getTitle(): TitleNode
         return $this->titleNode;
     }
 
-    public function addChild(DocumentEntryNode $child): void
+    public function addChild(DocumentEntryNode|ExternalEntryNode $child): void
```

Comment:
Please use `EntryNode` here. This should not break existing implementations.

Reply:
Cannot happyfy phpstan :-(